docs/src/main/tut/symbols.md

```diff
@@ -0,0 +1,19 @@
+| Symbol | Name                   | Typeclass      | Import                       | Source                                              |
+| ------ | ---------------------- | ------------   | --------------------------   | -------------------------------------------------   |
+| `<*>`  | apply                  | `Apply`        | TBA                          | core/src/main/scala/cats/Apply.scala                |
+| `|@|`  | Cartesian builder      | `Cartesian`    | `cats.syntax.cartesian._`    | core/src/main/scala/cats/Cartesian.scala            |
```

Comment:
Would it make sense to link to the scaladoc instead of the source? That should have the generated methods, and it contains a link to the source in the non-generated case.